packages/node_modules/@webex/react-redux-spark/src/sdk.js

```diff
@@ -142,3 +141,32 @@ export function createSDKGuestInstance(jwt, options = {}) {
 
   return webexSDKInstance.authorization.requestAccessTokenFromJwt({jwt}).then(() => webexSDKInstance);
 }
+
+/**
+ * Handles creating a Proxy that enables hi-jacking the appName/appVersion
+ * before a request is sent to server. For usage with widgets where an
+ * existing SDK instance is used.
+ *
+ * @param {Object} sdkInstance An instance of the Webex SDK
+ * @param {*} name The name of the app to use
+ * @returns {Promise<object>}
```

Comment:
You're returning a `Proxy` object, does this behave like a `Promise`?

Reply:
Proxy will pass thru to SDK instance unless key is 'config'

Reply:
I think my point is that you're not returning a Promise from this function

Reply:
Fixed